data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+				accept
+
+mission "First Contact: Ka'het: Quarg 1D"
+	name "Abandoned Ringworld"
+	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	landing
+	source "Querii Ka"
+	destination "Lagrange"
+	stopover "Kuwaru Efreti"
+	stopover "Alta Hai"
+	to offer
+		not "First Contact: Ka'het: Quarg 1A: offered"
+		not "First Contact: Ka'het: Quarg 1B: offered"
+		not "First Contact: Ka'het: Quarg 1C: offered"
+		has "First Contact: Quarg: offered"
+		has "First Contact: Hai: offered"
+		has "First Contact: Kuwaru Efreti: offered"
+	on offer
+		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."
+		conversation
+			`Due to your past experience, you expect crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
+			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
+				accept
+
+mission "First Contact: Ka'het: Quarg 2"
+	landing
+	invisible
+	source
+		planet "Alta Hai" "Lagrange" "Kuwaru Efreti"
+	to offer
+		or
+			has "First Contact: Ka'het: Quarg 1A: active"
+			has "First Contact: Ka'het: Quarg 1B: active"
+			has "First Contact: Ka'het: Quarg 1C: active"
+			has "First Contact: Ka'het: Quarg 1D: active"
+	on offer
+		conversation
+			`Do you want to ask the Quarg about the abandoned ringworld in the region of the Graveyard?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	Walking down the principal, alien-looking hall, you decide to stop one of the few Quarg you see there. "Hello, human. How may I be of service?" the Quarg says. You reply by asking him about the Graveyard and the ringworld they built there.`
+			`	"Ah, you encountered the remainders of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they always refrained from accepting our aid, and attempted to relocate at a greater distance from us, in worlds easier to maintain than the home they were reserved."`
+			choice
+				`	"Who were the hostile contacts with?"`
+				`	"What happened to the Builders?"`
+					goto ka'het
+			`	"Long ago, few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young and foolish race, and they foolishly decided to attack the Builders instead of learning from them. They were stronger any vessel the Builders possessed, but also few enough to be destroyed by the fleets patrolling the area."`
```

Comment:
Repetitive use of "foolish".
```suggestion
			`	"Long ago, few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young race who foolishly decided to attack the Builders instead of learning from them. Their ships were stronger than any vessel the Builders possessed at the time, but were also few enough in number to be destroyed by the Builders."`
```